# vision_robot/msg/PersonInfo.msg
string name
string id
float32 confidence
int32 bbox_x
int32 bbox_y
int32 bbox_width
int32 bbox_height
